en/SUMMARY.md

```diff
@@ -61,6 +61,7 @@
        * [v3.2 (Detailed)](releases/stable_v3.2_long.md)
     * [Daily Builds](releases/daily_builds.md)
     * [Daily Build New Features](releases/daily_build_new_features.md)
+      * [Video Overlay](FlyView/videoOverlay.md)
```

Comment:
Minor point, but the naming of this file is completely inconsistent. I always use lower case and underscore. Don always uses CamelCase - ie VideoOverlay.md. This is neither.

Reply:
oops, fixed it!